From: Bhumika Goyal <bhumirks@gmail.com>
Date: Sat, 19 Aug 2017 16:22:37 +0530
Subject: irqchip/gic: Make irq_chip const
Patch-mainline: v4.14-rc1
Git-commit: 73c4c37c7e6ecc3cf921d182b733c9322ebed117
References: FATE#323954

Make this const as it is only used in a copy operation.
Done using Coccinelle.

Signed-off-by: Bhumika Goyal <bhumirks@gmail.com>
Signed-off-by: Marc Zyngier <marc.zyngier@arm.com>
Acked-by: Alexander Graf <agraf@suse.de>
---
 drivers/irqchip/irq-gic.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/irqchip/irq-gic.c
+++ b/drivers/irqchip/irq-gic.c
@@ -415,7 +415,7 @@
 	chained_irq_exit(chip, desc);
 }
 
-static struct irq_chip gic_chip = {
+static const struct irq_chip gic_chip = {
 	.irq_mask		= gic_mask_irq,
 	.irq_unmask		= gic_unmask_irq,
 	.irq_eoi		= gic_eoi_irq,
